shell/platform/darwin/macos/framework/Source/FlutterEngineTest.mm

```diff
@@ -742,6 +742,16 @@ - (nonnull NSView*)createWithViewIdentifier:(int64_t)viewId arguments:(nullable
       [FlutterMethodCall methodCallWithMethodName:@"System.exitApplication"
                                         arguments:@{@"type" : @"cancelable"}];
 
+  // Always terminate when the binding isn't ready (which is the default).
+  triedToTerminate = FALSE;
```

Comment:
```suggestion
  triedToTerminate = NO;
```

Reply:
Looks like there are a few other places above with the same issue.

Reply:
Fixed.